docusaurus/docs/reactnative/common-content/contexts/image-gallery-context/set_images.mdx

```diff
@@ -0,0 +1,5 @@
+Setter for value [`images`](../../../contexts/image_gallery_context.mdx#images).
+
+| Type                                                                                                                                                                         |
+| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `(messageWithImages: [MessageType](https://github.com/GetStream/stream-chat-react-native/blob/master/package/src/components/MessageList/hooks/useMessageList.ts)[]) => void` |
```

Comment:
This needs to be imported in `docusaurus/docs/reactnative/contexts/image_gallery_context.mdx` too?

Reply:
Yes, since its a prop to imageGallery and previously it wasn't imported(don't know the reason for the same).

Reply:
even now it isnt imported then. Maybe you forgot to push that change?

Reply:
Oh, so basically image_gallery_context already has this change without import, if you check but yes since we have a file for `setImages` we can import the same. I am pushing it.